### ISSUE TITLE
α-renaming variable from another file is confusing

### ISSUE THREAD

**Issue by sorawee** [racket/racket#1977]
Consider a file `a.rkt` with the following content:

```
#lang racket
(require "b.rkt")
(format "~a" hello)
(println hello)
```

and `b.rkt` with the following content:

```
#lang racket
(provide (all-defined-out))
(define hello 'world)
```

In `a.rkt`, right click on any `hello`. An entry named `Rename "b.rkt"` will appear. This should show `Rename hello` instead.

After clicking `Rename "b.rkt"`, a dialog pops up. Enter `hello2`. The content of `a.rkt` becomes:

```
#lang racket
(require hello2)
(format "~a" hello2)
(println hello2)
```

which probably is not the most ideal outcome.

**Comment by sorawee** [comment]
This is a little bit worse than I thought. With the same setup:

```
#lang racket

; a.rkt

(require "b.rkt")
(format "~a" world)
(println hello)
```

```
#lang racket

; b.rkt

(provide (all-defined-out))
(define hello 1)
(define world 2)
```

After renaming `hello` to `hello2`:

```
#lang racket

; a.rkt

(require hello2)
(format "~a" hello2)
(println hello2)
```

**Comment by shhyou** [comment]
Fixed by @sorawee in racket/drracket#415
